src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const isMale = (sex) => {
+    return sex === 'm';
+  };
+
+  const men = century ? people.filter(
+    person => Math.ceil(person.died / 100) === century && isMale(person.sex))
+    : people.filter(person => isMale(person.sex));
```

Comment:
```suggestion
  const men = people.filter(
    person => person.sex === 'm',
  );

  const diedInCentury = century 
    ? men.filter(person => Math.ceil(person.died / 100) === century)
    : men;
```

в других функциях аналогично